common/kernel/timing.cc

```diff
@@ -566,12 +568,18 @@ void TimingAnalyser::walk_forward()
             auto &pd = ports.at(sp.first);
             DelayPair init_arrival(0);
             CellPortKey clock_key;
-            // TODO: clock routing delay, if analysis of that is enabled
             if (sp.second != IdString()) {
                 // clocked startpoints have a clock-to-out time
                 for (auto &fanin : pd.cell_arcs) {
                     if (fanin.type == CellArc::CLK_TO_Q && fanin.other_port == sp.second) {
-                        init_arrival = init_arrival + fanin.value.delayPair();
+                        init_arrival += fanin.value.delayPair();
+                        // Include the clock delay if clock_skew analysis is enabled
+                        if (with_clock_skew) {
+                            auto clock_delay = ports.at(CellPortKey(sp.first.cell, fanin.other_port)).route_delay;
+                            clock_delay.min_delay = clock_delay_fac * clock_delay.min_delay;
```

Comment:
The `clock_delay_fac` is just for debugging for me. Once this PR is approved I will remove it. It just makes seeing what delay is the clock skew and what isn't much easier in the reports :).